### PR TITLE
Handle void element.

### PR DESCRIPTION
**Pull request by paulo-silva**
Given

```heex
<div>
<link rel="shortcut icon" href={Routes.static_path(@conn, "/images/favicon.png")} type="image/x-icon">
<p>some text</p>
<br>
<hr>
<input type="text" value="Foo Bar">
<img src="./image.png">
</div>
```

Expected

```heex
<div>
  <link
    rel="shortcut icon"
    href={Routes.static_path(@conn, "/images/favicon.png")}
    type="image/x-icon"
  />
  <p>some text</p>
  <br />
  <hr />
  <input type="text" value="Foo Bar" />
  <img src="./image.png" />
</div>
```

Address #31

### CODE REVIEW

File: lib/heex_formatter/formatter.ex [hunk 1]
```diff
@@ -68,6 +68,31 @@ defmodule HeexFormatter.Formatter do
     |> then(&(&1 <> "\n"))
   end
 
+  @void_tags ~w(br hr input img link meta)
```
Comment (feliperenan) [inline lib/heex_formatter/formatter.ex]:
```suggestion
  @void_tags ~w(area base br col hr img input link meta param command keygen source)
```

**Review by feliperenan — APPROVED** [review]
Looks great ✨

**Comment by paulo-silva** [comment]
closes #33

**Comment by paulo-silva** [comment]
Closes #31

**Pull request merged**